port/win/port_win.h

```diff
@@ -241,6 +241,17 @@ extern void InitOnce(OnceType* once, void (*initializer)());
 #define CACHE_LINE_SIZE 64U
 #endif
 
+// To reduce false cache sharing by enforcing alignment
+#if defined(__clang__) || defined(__GNUC__)
+# define CACHE_ALIGNED(size) alignas(size)
```

Comment:
The placement of alignas is proper. However, just like `#pragma` can not be within a `#define` definition w/o `__Pragma` `alignas` requires as special keyword `_Alignas` for being within `#define`.

Therefore, the proper syntax is `#define CACHE_ALIGNED(size) _Alignas(size)`.

However, since `alignas` is a part of C++11 I suggest we do not put anything in port until someone needs it and just use it.